src/desktop/file_chooser.rs

```diff
@@ -125,6 +125,31 @@ impl FileFilter {
     }
 }
 
+#[cfg(feature = "backend")]
+impl FileFilter {
+    pub fn label(&self) -> &str {
+        &self.0
+    }
+
+    pub fn mimetype_filters(&self) -> Vec<&str> {
+        self.1
+            .iter()
+            .filter_map(|(type_, string)| {
+                (type_ == &FilterType::MimeType).then_some(string.as_str())
+            })
+            .collect()
+    }
+
+    pub fn pattern_filters(&self) -> Vec<&str> {
```

Comment:
```suggestion
    pub fn patterns(&self) -> Vec<&str> {
```

Reply:
We need to change the api here https://docs.rs/ashpd/latest/ashpd/desktop/file_chooser/struct.FileFilter.html first otherwise its a bit confusing.